apps/design/backend/src/worker/index.ts

```diff
@@ -0,0 +1,30 @@
+import { ensureDirSync } from 'fs-extra';
+import path from 'path';
+import { assertDefined } from '@votingworks/basics';
+
+import { WORKSPACE } from '../globals';
+import { Store } from '../store';
+import * as worker from './worker';
+
+async function main(): Promise<void> {
+  const workspacePath = path.resolve(assertDefined(WORKSPACE));
+  ensureDirSync(workspacePath);
+  const dbPath = path.join(workspacePath, 'design-backend.db');
+  const store = Store.fileStore(dbPath);
```

Comment:
As far as I can tell, this is the first case of two processes accessing the same SQLite DB in VxSuite. I did verify that this is okay:

> Multiple processes can have the same database open at the same time. Multiple processes can be doing a SELECT at the same time. But only one process can be making changes to the database at any moment in time, however.
>
> Reference: https://www.sqlite.org/faq.html

We may run into some quirks with simultaneous writes by the server and the worker, but I wasn't able to trigger any such errors with my manual testing. I _believe_ `better-sqlite3` automatically retries when it finds that the DB is busy. Regardless, when we deploy using Heroku, we'll be using Postgres, rendering most of these concurrency concerns irrelevant.